4094 add info message to the edit HTTL popup

venetrius opened the pull request:
related to: https://github.com/camunda/camunda-bpm-platform/issues/4094

danielkelemen reviewed (changes requested):
Looks pretty good. I added a few suggestions.

❌ There is also a prettier error on the CI.

venetrius commented on webapps/frontend/public/app/cockpit/locales/en.json:
```diff
@@ -2491,6 +2491,10 @@
     "TIME_TO_LIVE_POPUP_RADIO_CLEAR_TOOLTIP": "Sets the value to null",
     "TIME_TO_LIVE_POPUP_RADIO_UPDATE": "Update Value",
     "TIME_TO_LIVE_POPUP_RADIO_UPDATE_TOOLTIP": "Updates the value with the provided one",
+    "TIME_TO_LIVE_POPUP_REMOVAL_TIME_NOT_UPTDATED_INFO": "Updating the History Time To Live value does not affect instances that have already been completed. ",
```
Thanks, updated. Added spell checker to my IDE.

venetrius commented on webapps/frontend/public/app/cockpit/locales/en.json:
```diff
@@ -2491,6 +2491,10 @@
     "TIME_TO_LIVE_POPUP_RADIO_UPDATE_TOOLTIP": "Updates the value with the provided one",
+    "TIME_TO_LIVE_POPUP_REMOVAL_TIME_NOT_UPTDATED_INFO": "Updating the History Time To Live value does not affect instances that have already been completed. ",
+    "TIME_TO_LIVE_POPUP_RUN_BATCH_JOB_INFO_START": "Run a ",
+    "TIME_TO_LIVE_POPUP_RUN_BATCH_JOB_INFO_LINK": "Batch Operation ",
```
I really did not need those spaces

venetrius commented on webapps/frontend/ui/cockpit/client/scripts/directives/time-to-live/time-to-live-dialog.js:
```diff
@@ -27,6 +27,7 @@ const Controller = [
   '$translate',
   function(camAPI, $modalInstance, $scope, Notifications, $translate) {
     const resource = camAPI.resource($scope.resource);
+    $scope.showLinkToBachProcess = $scope.$parent?.resource !== "case-definition";
```
Updated

venetrius commented:
> Looks pretty good. I added a few suggestions.
> 
> ❌ There is also a prettier error on the CI.

Thanks! Made suggested changes
(also added spellChecker and ESLint to my IDE 🤦 )
